trulens/nn/models/pytorch.py

```diff
@@ -77,40 +78,18 @@ def __init__(
             model.eval()
 
         if device is None:
-            device = pytorch.get_default_device()
-
-            devices = set(p.device for p in model.parameters())
-
-            if len(devices) > 1:
-                tru_logger.warning(
-                    f"Model's parameters span more than one device ({devices})."
-                )
-
-            if len(devices) == 0:
-                # Model without any parameters. No need to do anything here.
-                pass
-            else:
-                mdevice = list(devices)[0]
-
-                if mdevice != device:
-                    tru_logger.warning(
-                        f"Model is not on default device ({device}), moving it there. "
-                        f"If you intend to work on {mdevice}, set it as the default pytorch device or explicitly provide it as the device argument to get_model_wrapper."
-                    )
-                    model.to(device)
-
-        else:
-            model.to(device)
-
-            def_device = pytorch.get_default_device()
-            if device != def_device:
-                tru_logger.warning(
-                    f"Model's device ({device}) differs from pytorch's default device ({def_device}). Changing default to model device."
+            try:
+                device_counter = Counter(
```

Comment:
Maybe more of an edge case but how do we want to deal with models living on multiple GPUs (whether to parallelize, or not enough memory to fit on one GPU)? Is it feasible to skip the `model.to(self.device)` and keep the model as-is?

Reply:
I thought about this and I think we should cross that bridge when we get to it.
There's the more common case that they never actually specify their model device, and we end up using some default. of which the model.to is useful in fixing a user mistake which would otherwise fail.

I think since that is a more complex scenario, that should require more deliberation by the user, so we may add a "skip device defaulting" option in the future.